test/python/test_wrapper.py

```diff
@@ -92,6 +93,28 @@ def test_unregister_non_existent(self):
             qiskit.wrapper.unregister('provider1')
         self.assertEqual(initial_providers, registered_providers())
 
+    @requires_qe_access
+    def test_register_backend_name_conflicts(self, QE_TOKEN, QE_URL,
+                                             hub, group, project):
+        """Test backend name conflicts when registering."""
+
+        qiskit.wrapper.register(QE_TOKEN, QE_URL, hub, group, project,
+                                provider_name='provider1')
+        initial_providers = registered_providers()
+        initial_backends = qiskit.wrapper.available_backends()
+        with self.assertLogs(level=logging.WARNING) as context:
```

Comment:
Can we rename `context` with `logger_output` or simply `output`? Does it make sense?